src/libretro/Makefile

```diff
@@ -107,8 +107,10 @@ endif
 LDFLAGS += $(LIBM)
 
 ifeq ($(DEBUG), 1)
+   CFLAGS += -O0 -g
    CXXFLAGS += -O0 -g
 else
+   CFLAGS += -O3
```

Comment:
Didn't cause any issues over here, at least nothing that can be tested on a short time window. Usually i'd go for O2 instead since it is less aggressive and is considered safe enough for the Linux Kernel, but given that this is a project with a FAR smaller scope, O3 should be fine here going forward.